src/e2e/admin/bots.test.ts

```diff
@@ -112,4 +116,30 @@ describe('Admin - Bot Management', () => {
     await expectAdminApiCallSuccess(`workspace/bots/${tempBotId}/delete`, 'POST')
     await page.waitFor(200)
   })
+
+  it('Changes bot converse config to disable public endpoint', async () => {
+    await clickOn('#btn-menu-code-editor') // Navigate to admin code editor
+    await page.waitFor(1000) // Wait  for code editor to display
+    await clickOn('span.bp3-button-text', { text: 'Advanced Editor' }) //Display raw editor
```

Comment:
This shouldn't be necessary. Botpress config is available without the advanced editor. Is there something hindering the test without it ?

Reply:
It's bot config not botpress.config. is there an issue ? it's a test and it's working I don't want to spend more time on this

Reply:
Ah right. Nope, didn't see it was in admin and not studio.